zio-kafka/src/main/scala/zio/kafka/utils/SslHelper.scala

```diff
@@ -40,13 +43,10 @@ object SslHelper {
                                     ZIO.attempt(SocketChannel.open(addr))
                                   )(channel => ZIO.attempt(channel.close()).orDie)
                        tls <- ZIO.attempt {
```

Comment:
🌕 I'm wondering if we shouldn't use `attemptBlocking` here. The `channel.read(buf)` is most likely blocking while waiting for/receiving the response, no? 🤔

Same for the `SocketChannel.open(addr)` and `channel.close()` call above, these Java functions are most likely blocking IMO. 🤔

Reply:
The code is already embedded in a `ZIO.blocking` block.

Reply:
My bad. Thanks!